integrations.yml

```diff
@@ -11,6 +11,12 @@
 # 8 - Contracts and Proposals
 # 9 - Developer-only Tools
 #
+-
+  name: "CozyCal Scheduling"
+  description: "Allow your guests to book support calls, video meetings, rooms, and more. New events are automatically added your Basecamp schedule."
+  website: "https://www.cozycal.com/?ref=basecamp"
```

Comment:
can you please change this to the page you linked in https://github.com/basecamp/bc3-integrations/pull/191#issuecomment-1154486375